atom/browser/native_window_views.cc

```diff
@@ -10,7 +10,6 @@
 
 #include <string>
 #include <vector>
-
```

Comment:
This empty line should be kept, see:
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Names_and_Order_of_Includes